libraries/matware/jupgrade/schemas/1.5/contents.php

```diff
@@ -179,8 +179,8 @@ public function dataHook($rows = null)
 			// @@ Duplicate entry for key 'idx_client_id_parent_id_alias_language'
 			if ($content->load(array('alias' => $row['alias'], 'catid' => $row['catid'])))
 			{
-				// Getting the duplicated alias
-				$alias = $this->getAlias('#__content', $row['alias']);
+				$content->reset();
+				$content->id = 0;
 				// Set the modified alias
 				$row['alias'] .= "-".rand(0, 99999999);
```

Comment:
To test this (or examle);

Old content table:
id 1, alias test
id 2,alias test

In the migrated content table you will see only:
id 1, alias test-1324123 
and there will be no second record as wrongly id 1 got updated.

After my change you will get
id 1, alias test
id 2, alias test-889494